src/Chainweb/Pact/SPV.hs

```diff
@@ -149,14 +149,3 @@ getTxIdx bdb pdb bh th = do
 
     sindex :: Monad m => (a -> Bool) -> S.Stream (S.Of a) m () -> m (Maybe Natural)
     sindex p s = S.zip (S.each [0..]) s & sfind (p . snd) & fmap (fmap fst)
-
--- -------------------------------------------------------------------------- --
--- utilities
-
--- | Prepend "spvSupport" to any errors so we can differentiate messages
---
-spvError :: Text -> IO (Either Text a)
-spvError = return . Left . (<>) "spvSupport: "
-
-spvError' :: String -> IO (Either Text a)
-spvError' = spvError . pack
```

Comment:
The error strings already contain SPV data that looks like `SPV verify failed: <errstring>` coming from Pact. These functions are duplicate that work, so they're unnecessary.